tests/XFTPCLI.hs

```diff
@@ -55,6 +56,8 @@ testXFTPCLISendReceive = withXFTPServer $ do
   testInfoFile fdRcv2 "Recipient"
   testReceiveFile fdRcv2 "testfile_1" file
   testInfoFile fdSnd "Sender"
+  xftp ["recv", fdSnd, recipientFiles, "--tmp=tests/tmp"]
+    `shouldThrow` anyException
```

Comment:
It’s better to check for specific exception maybe?
Also - why would CLI have throw an exception at all - should it not print error?

Reply:
when run as an executable it prints a correct error, then exits with an error code. in tests trying to check for output fails with "uncaught exception: ExitCode", probably due to how capture_ / withArgs work